content/1.introduction/1.getting-started.md

```diff
@@ -1,3 +1,4 @@
+<img align="left" src="https://docs.sillylittle.tech/slfdocs.png" height="200" width="200" alt="badge"/> <br />
```

Comment:
Use a local operator instead of manually referencing the file so that way it can show in preview.
This should also be fixed on https://github.com/dswan36/ezprox/pull/9 and https://github.com/dswan36/acprox/pull/51